### PR TITLE
Introduced lerna package to manage multiple packages 

### PR DESCRIPTION
**Pull request by anhtran-ccpo**
- Added package [Lerna](https://github.com/lerna/lerna) - _A tool for managing JavaScript projects with multiple packages._
- Moved all API projects and their microservice implementations to new `./packages` directory
- Consolidated and moved file `.eslintrc.json` to the root of the repository for use as central configuration
- Added new file `package.json` to the root of the repository that uses [lerna commands](https://lerna.js.org/#commands)
- Workflows such as linting and testing can now be executed from the repository root for multiple packages

Ticket: AT-6403

### CODE REVIEW

**Comment by jasonburkert-ccpo** [comment]
I want to get this in, but I didn't want to cause any chaos as we approach the end of the sprint.

File: packages/random_quote/shared_infrastructure/main.tf [hunk 1]
```diff
@@ -37,7 +37,7 @@ resource "azurerm_storage_account" "sa" {
 }
 
 module "keyvault" {
-  source              = "./modules/keyvault"
+  source              = "modules/keyvault"
```
Comment (anhtran-ccpo) [inline packages/random_quote/shared_infrastructure/main.tf:40]: Note to self: change this back

File: packages/provisioning/shared_infrastructure/main.tf [hunk 2]
```diff
@@ -37,7 +37,7 @@ resource "azurerm_storage_account" "sa" {
 }
 
 module "keyvault" {
-  source              = "./modules/keyvault"
+  source              = "modules/keyvault"
```
Comment (anhtran-ccpo) [inline packages/provisioning/shared_infrastructure/main.tf:40]: Note to self change this back

Comment (anhtran-ccpo) [reply packages/provisioning/shared_infrastructure/main.tf]: Changed

File: packages/random_quote/shared_infrastructure/main.tf (hunk 1 above)
Comment (anhtran-ccpo) [reply packages/random_quote/shared_infrastructure/main.tf]: Changed

File: package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "root",
+  "private": true,
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
+    "lint": "lerna run lint",
+    "lint:fix": "lerna run lint:fix",
+    "test": "lerna run test --parallel"
```
Comment (jasonburkert-ccpo) [inline package.json:8]: Where the magic happens.

**Review by jasonburkert-ccpo — APPROVED** [review]
Ran the following from root of repository. Looks good.
```
npm ci
npm run bootstrap
npm run lint
npm run test
```

File: package.json (hunk 3 above)
Comment (anhtran-ccpo) [reply package.json:8]: I think the lint commands can be done in parallel

**Review by jeffsegal-ccpo — APPROVED** [review]

**Pull request merged**